### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by VarshaSaroj**

### CODE REVIEW

**Review by saurabhanilpandey — APPROVED** [review]
Congratulations on completing task 1

File: Varsha2001/Task1.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+HTML elements label pieces of content such as "this is a heading", "this is a paragraph", "this is a link", etc.</p></div>
+<p>Web pages can be created and modified by using professional HTML editors.
+
+However, for learning HTML we recommend a simple text editor like Notepad (PC) or TextEdit (Mac).
+
+We believe that using a simple text editor is a good way to learn HTML.</p>
+<hr>
+<p>Here is a song video linked with my page.</p>
+<hr>
+<h2>-:List:-</h2>
```
Comment (PradyumnaKrishna) [inline Varsha2001/Task1.html:24]: This looks good however, you can remove this code as it is not required in task.

**Review by PradyumnaKrishna — CHANGES_REQUESTED** [review]
I am sorry but the folder name is still incorrect. It should be `VarshaSaroj` instead of `Varsha2001`.

Please fix this as I won't be able to merge this with incorrect folder name.

**Comment by PradyumnaKrishna** [comment]
I would also suggest you to push your code to branch `patch-1` and reopen #3 as it was your first Pull Request. Closing this one and merging #3.

**Comment by PradyumnaKrishna** [comment]
Closing (duplicate of #3)

**Pull request closed without merging**